### PR TITLE
Updates tests according to switching flow changes

### PR DESCRIPTION
**Pull request by dev-gh**
Now switching is being triggered externally so new signals handling has
been addes and tests have been changed accordingly.

### CODE REVIEW

File: test_scripts/iAP2TransportSwitch/common.lua [hunk 1]
```diff
@@ -27,29 +27,28 @@ m.device = {
     id = "127.0.0.1",
     port = 23456,
     out = "iap2bt.out",
-    type = "BLUETOOTH"
+    type = "BLUETOOTH",
```
Comment (dboltovskyi) [inline test_scripts/iAP2TransportSwitch/common.lua]: @dev-gh Please remove unused module 'hmi_values' in line 17

File: test_scripts/iAP2TransportSwitch/common.lua [hunk 2]
```diff
@@ -94,6 +93,25 @@ function module:connectMobile(pDevice)
   return module:expectEvent(events.connectedEvent, "Connected", pDevice)
 end
 
+function module:doTransportSwitch(device)
+  input = io.open(config.pathToSDL.."/iap_signals_in", "w")
```
Comment (dboltovskyi) [inline test_scripts/iAP2TransportSwitch/common.lua]: @dev-gh Please use local variable here and below

Comment (dev-gh) [reply test_scripts/iAP2TransportSwitch/common.lua]: @dboltovskyi 
Done with 24aad0b

File: test_scripts/iAP2TransportSwitch/common.lua (hunk 1 above)
Comment (dev-gh) [reply test_scripts/iAP2TransportSwitch/common.lua]: @dboltovskyi 
Done with 24aad0b

**Pull request merged**